packages/flutter/lib/src/material/text_selection_theme.dart

```diff
@@ -16,6 +17,11 @@ import 'theme.dart';
 /// Use [TextSelectionTheme.of] to access the closest ancestor
 /// [TextSelectionTheme] of the current [BuildContext].
 ///
+/// Note, on iOS [TextField] and [SelectableText] cannot access
+/// [TextSelectionThemeData.selectionHandleColor]. To set the
+/// `selectionHandleColor` you can change the [CupertinoThemeData.primaryColor]
+/// in [ThemeData.cupertinoOverrideTheme].
+///
```

Comment:
Would these docs be better on the `selectionHandleColor` property instead of on `TextSelectionThemeData` itself?

Reply:
Make sense, I'll do the change.